resource/lvm/lowlevel/exec.go

```diff
@@ -0,0 +1,109 @@
+package lowlevel
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+	"syscall"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/pkg/errors"
+)
+
+type Exec interface {
+	Run(prog string, args []string) error
+	RunExitCode(prog string, args []string) (int, error) // for mountpoint querying
+	Read(prog string, args []string) (stdout string, err error)
+	ReadWithExitCode(prog string, args []string) (stdout string, rc int, err error) // for blkid querying
+
+	// unit read/write injection
+	ReadFile(fn string) ([]byte, error)
+	WriteFile(fn string, c []byte, p os.FileMode) error
+	MkdirAll(path string, perm os.FileMode) error
+	Exists(path string) (bool, error)
+}
+
+type OsExec struct {
+}
+
+func (*OsExec) Run(prog string, args []string) error {
+	log.WithField("module", "lvm").Infof("Executing %s: %v", prog, args)
```

Comment:
We are trying to avoid using log. This will not be displayed to the user, so you should use status or an error. If you do need to log, use debug level. Please fix throughout.